routes/web.php

```diff
@@ -12,5 +12,22 @@
 */
 
 $router->get('/', function () use ($router) {
-    //return $router->app->version();
+    return 'API v1';
+});
+
+$router->group(['prefix' => 'api/v1'], function () use ($router) {
+	
+	//--=API DIR START=--
+    $router->get('direction', 'DirectionController@get');
```

Comment:
Name of routes `directions/...` can be logical instead of `direction/...`.
But it's up to you

Reply:
Fixed